PolyEngine/Engine/Src/TimeSystem.cpp

```diff
@@ -0,0 +1,60 @@
+#include "EnginePCH.hpp"
+
+#include "TimeSystem.hpp"
+#include <chrono>
+
+using std::chrono::steady_clock;
+using std::chrono::duration_cast;
+using std::chrono::duration;
+
+//------------------------------------------------------------------------------
+void Poly::TimeSystem::TimeUpdatePhase(World * world)
+{
+	TimeWorldComponent* timeComponent = &world->GetTimeWorldComponent();
+
+	//Get new delta time
+	std::chrono::steady_clock::time_point frameTime = steady_clock::now();
+	duration<float> deltaTime = duration_cast<duration<float>>(frameTime - timeComponent->LastFrameTime);
+#ifdef DEBUG
```

Comment:
in future this should be surrounded with ifndef FINAL. As of right now, please remove it and add propper comment.